src/site/content/en/metrics/fid/index.md

```diff
@@ -196,6 +207,24 @@ How you track, report on, and analyze FID will probably be quite a bit different
 from other metrics you may be used to. The next section explains how best to do
 this.
 
+### Why only consider the input delay?
+
+As mentioned above, FID only measures the "delay" in event processing. It does
+not measure the event processing time itself nor the time it takes the browser
+to update the UI after running event handlers.
+
+Even though this time is important to the user and does affect the experience,
+it's not included in this metric because it would be easy for developers to
```

Comment:
I think we should phrase this in terms of what improving the metric incentivizes, not what developers can do to 'cheat'. Just a wording suggestion.

Reply:
Updated.